ui/oopsyraidsy/data/04-sb/trial/shinryu.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+// Shinryu Normal
+
+[{
+  zoneRegex: {
+    en: /^The Royal Menagerie$/,
+    cn: /^阿拉米格王宫屋顶庭园$/,
+    ko: /^알라미고 왕궁 옥상정원$/,
+  },
+  damageWarn: {
+    'Akh Rhai': '1FA6', // Sky lasers alongside Akh Morn.
+    'Blazing Trail': '221A', // Rectangle AoEs, intermission adds.
+    'Collapse': '2218', // Circle AoEs, intermission adds
+    'Dragonfist': '24F0', // Giant punchy circle in the center.
+    'Earth Breath': '1F9D', // Conal attacks that aren't actually Earth Shakers.
+    'Gyre Charge': '1FA8', // Green dive bomb attack.
+    'Spikesicle': '1FA`', // Blue-green line attacks from behind.
+    'Tail Slap': '1F93', // Red squares indicating the tail's landing spots.
+  },
+  triggers: [
+    {
+      id: 'Shinryu Normal Levinbolt',
+      damageRegex: '1F9C',
+      condition: function(e) {
+        // Double taps only
+        return e.type != '15';
+      },
+      mistake: function(e) {
+        return { type: 'warn', blame: e.targetName, text: e.abilityName };
+      },
+    },
+    {
+      // Icy floor attack.
+      id: 'Shinryu Normal Diamond Dust',
+      gainsEffectRegex: gLang.kEffect.ThinIce,
+      deathReason: function(e) {
+        return { type: 'fail', name: e.targetName, reason: { en: 'Slid off!' } };
```

Comment:
We really need to find some network data for falling to your death.  Some day~

Reply:
Someday.